.github/workflows/push.yml

```diff
@@ -9,7 +9,10 @@ env:
   TEST_KBC_STORAGE_API_TOKEN_EXPIRED: ${{ secrets.TEST_KBC_STORAGE_API_TOKEN_EXPIRED }}
 jobs:
   Build:
-    concurrency: ci # only one parallel job allowed - used shared testing project
+    # only one parallel job allowed - used shared testing project
+    concurrency:
+      group: ci
+      cancel-in-progress: false
```

Comment:
Neviem, ci zmenili default hodnotu, alebo je tam nejaky bug, ... ale novsi build mi zrusil starsi, co nechcem.
Tak to skusim explicitne nastavit.